### PR TITLE
Use PPA for all releases

### PR DESCRIPTION
**Pull request by SlothOfAnarchy**

### CODE REVIEW

File: tasks/install.yml [hunk 1]
```diff
@@ -3,31 +3,32 @@
...
+      cosmic: cosmic
+      devel: devel
+  register: ppa
 
-- name: Install ZFS package
+- name: Install zfsutils-linux
   apt:
     name: zfsutils-linux
 
-- name: Add zfs module to /etc/modules
```
Comment (Scriptkiddi) [inline tasks/install.yml:17]: Is this now done by the package?

File: tasks/install.yml [hunk 2]
```diff
@@ -3,31 +3,32 @@
...
-    line: 'zfs'
-    state: present
-  register: kernelmodules
-
 - name: Configure ZFS kernel parameters
   template:
     src: zfs.conf.j2
     dest: /etc/modprobe.d/zfs.conf
 
-- name: Reload kernel modules
```
Comment (Scriptkiddi) [inline tasks/install.yml:29]: Is this now done by the package?

File: tasks/install.yml (hunk 1 above)
Comment (SlothOfAnarchy) [reply tasks/install.yml:17]: It is not necessary anymore. I removed this line and rebooted a ZFS server without any problems.

File: tasks/install.yml (hunk 2 above)
Comment (SlothOfAnarchy) [reply tasks/install.yml:29]: It is not necessary anymore. I removed this line and rebooted a ZFS server without any problems.

**Review by dasJ — APPROVED** [review]

**Pull request merged**